modules/vpc-scenario-3/main.tf

```diff
@@ -56,19 +54,22 @@ module "private-subnets" {
   azs         = "${var.azs}"
   vpc_id      = "${module.vpc.vpc_id}"
   public      = false
-  name_prefix = "${var.name_prefix}-private"
+  name_prefix = "${var.name_prefix}-vpn"
   cidr_blocks = "${var.private_subnet_cidrs}"
   extra_tags  = "${var.extra_tags}"
 }
 
-module "nat-gateway" {
-  source             = "../nat-gateways"
-  vpc_id             = "${module.vpc.vpc_id}"
-  name_prefix        = "${var.name_prefix}"
-  nat_count          = "${length(module.private-subnets.ids)}"
-  public_subnet_ids  = ["${module.public-subnets.ids}"]
-  private_subnet_ids = ["${module.private-subnets.ids}"]
-  extra_tags         = "${var.extra_tags}"
+# Nat Public/VPN Gateways
```

Comment:
Why is the module being swapped with this implementation? I was under the impression your topology requirements had removed the NAT, so seeing it here is confusing as a reviewer. How does this differ from what is provided by the module? Why was the module not ok to use? Did you consider updating the module? I don't see any notes about this or the decisions here in the ticket. This information should be more readily available and easier for me to confirm and validate.